src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php

```diff
@@ -1926,6 +1926,15 @@ private function getPaymentErrorMessages(Exception $e)
                 'Only numbers and decimal points (".") are allowed in the amount fields of the payment block, e.g. 10.50 or 1050.',
                 'Admin.Orderscustomers.Notification'
             ),
+            OrderConstraintException::class => [
```

Comment:
Maybe using php's `sprintf` method would be more readable ?

Sorry for the indentation, but something like this:

```php
OrderConstraintException::class => [
    OrderConstraintException::INVALID_PAYMENT_METHOD =>
        sprintf(
            '%s %s ^<>={}',
            $this->trans(
                'The selected payment method is invalid.',
                'Admin.Orderscustomers.Notification'
            ),
            $this->trans(
                'Invalid characters:',
                'Admin.Notifications.Info'
            )
        )
],

```

Reply:
It would be nicer indeed